content/getting-started/what-is-blazor.md

```diff
@@ -38,6 +38,10 @@ JavaScript is a powerful language but it has its disadvantages. Some are fixed b
 
 A prototype for statically compiling managed code into one [*.wasm* file](https://developer.mozilla.org/en-US/docs/WebAssembly/Text_format_to_wasm) [already exists](http://www.mono-project.com/news/2018/01/16/mono-static-webassembly-compilation/). It is possible, if not likely, that Blazor will move away from interpreting IL towards the statically compiled model over time.
 
+The following images illustrates the overall architecture of Blazor.
+
+![Blazor Architecture](/images/getting-started/blazor-architecture.jpg)
```

Comment:
The whole learn-blazor.com site is under [MIT license](https://github.com/software-architects/learn-blazor/blob/master/LICENSE). All the content should be open source and/or under Creative Commons license. The image contains a copyright notice. Would you be ok to remove the copyright or publish it under a CC license?